testing/common_testing_util.py

```diff
@@ -22,6 +32,10 @@
 path_data_functional_source: str = str(path_repo_root / "data_functional_testing")
 path_data_functional_tmp: str = str(path_repo_root / "tmp" / Path(path_data_functional_source).name)
 
```

Comment:
Maybe `Path` as well.

Reply:
So this raises a good point. I left these as str as we probably should issue PR to update all these to utilize path properly. 

Another good point from downstream function perspective is always call "Path())" on the input to allow flexible STR or PATH as parameters.